packages/ui/libs/user/view/src/pages/LoginPage/LoginPage.effects.ts

```diff
@@ -1,13 +1,27 @@
-import { useUserLogInFormConfig } from '@energyweb/origin-ui-user-logic';
+import {
+  useUserLogInFormConfig,
+  TUserLoginFormValues,
+  INITIAL_FORM_VALUES,
+} from '@energyweb/origin-ui-user-logic';
 import { useUserLogin } from '@energyweb/origin-ui-user-data';
 import { useNavigate } from 'react-router';
 import { UserModalsActionsEnum, useUserModalsDispatch } from '../../context';
 import { InvitationDTO } from '@energyweb/origin-backend-react-query-client';
+import { useState } from 'react';
 
 export const useLogInPageEffects = () => {
   const navigate = useNavigate();
   const dispatchModals = useUserModalsDispatch();
 
+  const [formValues, setFormValues] =
+    useState<TUserLoginFormValues>(INITIAL_FORM_VALUES);
+
+  const onWatchHandler = (values: string[]) => {
```

Comment:
To be honest, I don't think this is the best way to achieve the desired behaviour of the form, but we'll keep it for now. Probably will dig into validation modes and pick the most suitable for our case.